crates/stdarch-verify/tests/x86-intel.rs

```diff
@@ -155,7 +156,7 @@ fn verify_all_signatures() {
     //   https://software.intel.com/sites/landingpage/IntrinsicsGuide/#
     //
     // Open up the network console and you'll see an xml file was downloaded
-    // (currently called data-3.4.xml). That's the file we downloaded
+    // (currently called data-3.6.xml). That's the file we downloaded
```

Comment:
Ignoring pack files this update will grow the repo size by 6.74 MB. With pack files, it will likely be a bit smaller, though possibly not by much.

Reply:
Good point, I did not consider the size. If that is a concern I could revert and rebase adding this file. It could be added once there are actual plans to implement to implement for example the VNNI or FP16 extensions. My goal is to work on avx512f and avx512vl for now, especially masked loads/stores.